packages/xo-web/src/common/intl/messages.js

```diff
@@ -2593,6 +2593,8 @@ const messages = {
   pifsNotAttached: 'Not all PIFs are attached',
   pifsNotStatic: 'Not all PIFs are static',
   replication: 'Replication',
+  restartRequiredWarningMessage:
+    'Hosts toolstask will restart if packages need to be installed. Do you want to continue?',
```

Comment:
What do you think of: `If packages need to be installed, the toolstack on those hosts will restart. Do you want to continue?`